### PR TITLE
executor: remove Next function for SetExec

### PR DESCRIPTION
**Pull request by yangwenmai**

### CODE REVIEW

**Comment by XuHuaiyu** [comment]
LGTM

File: executor/set.go [hunk 1]
```diff
@@ -42,19 +42,6 @@ type SetExecutor struct {
...
-	err := e.executeSet()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	e.done = true
-	return nil, nil
-}
-
 // NextChunk implements the Executor NextChunk interface.
 func (e *SetExecutor) NextChunk(ctx context.Context, chk *chunk.Chunk) error {
```
Comment (zz-jason) [inline executor/set.go]: `executeSet` can be removed and put the code logical into `NextChunk`

Comment (XuHuaiyu) [reply executor/set.go]: @yangwenmai Please address this comment.

File: executor/set.go [hunk 2]
```diff
@@ -42,26 +42,60 @@ type SetExecutor struct {
...
 	chk.Reset()
 	if !e.done {
```
Comment (zz-jason) [inline executor/set.go]: we can rewrite this function like this to improve the code readability. 
```go
if e.done {
    return nil
}
e.done = true
...
```

**Pull request merged**